protos/flyteidl/plugins/sagemaker.proto

```diff
@@ -0,0 +1,102 @@
+syntax = "proto3";
+
+package flyte.plugins.sagemaker;
+
+option go_package = "github.com/kumare3/awsflyteplugins/gen/pb-go/proto";
```

Comment:
Hmmm I think you should follow the regular path here... of other plugins...

`option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/plugins";`

Reply:
Since this has a lot of messages, can you actually add one extra level of nesting? 
`option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/plugins/sagemaker";`

If this works, can you also remove the SageMaker prefix of the messages below?

Reply:
sure let me give it a try